src/client/pkg/require/require.go

```diff
@@ -76,6 +76,46 @@ func OneOfEquals(tb testing.TB, expected interface{}, actuals []interface{}, msg
 	}
 }
 
+// OneOfEqualsString checks one string element of a slice equals a value.
+func OneOfEqualsString(tb testing.TB, expected string, actuals []string, msgAndArgs ...interface{}) {
+	equal := false
+	for _, actual := range actuals {
+		if expected == actual {
+			equal = true
+			break
+		}
+	}
+	if !equal {
+		fatal(
+			tb,
+			msgAndArgs,
+			"Not equal : %#v (expected)\n"+
+				" one of  != %#v (actuals)", expected, actuals)
+	}
+}
```

Comment:
It's kind of aesthetic, but if you pass a `[]string` to a function that expects a `[]interface{}` you get an error. You can't convert it either, so you have to `make` a new `[]interface{}` and copy all of the elements